roachtest: activerecord failed

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/17895302?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/17895302?buildTab=artifacts#/activerecord) on master @ [cea3ff5562160a3bf2802da052da2aaa40e1ccc1](https://github.com/cockroachdb/cockroach/commits/cea3ff5562160a3bf2802da052da2aaa40e1ccc1):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.1.0-alpha.00000000-dev-cea3ff5562160a3bf2802da052da2aaa40e1ccc1
Tests run against activerecord 7.2.1
8487 Total Tests Run
8486 tests passed
1 test failed
60 tests skipped
5 tests ignored
22 tests passed unexpectedly
0 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- PASS: CompatibilityTest7_0#test_options_are_not_validated - unknown (unexpected)
--- PASS: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date - unknown (unexpected)
--- PASS: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array - unknown (unexpected)
--- PASS: CompatibilityTest6_0#test_datetime_doesnt_set_precision_on_change_column - unknown (unexpected)
--- PASS: CompatibilityTest5_2#test_datetime_doesnt_set_precision_on_change_column - unknown (unexpected)
--- PASS: CompatibilityTest5_2#test_options_are_not_validated - unknown (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- PASS: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default - unknown (unexpected)
--- PASS: CompatibilityTest6_1#test_datetime_doesnt_set_precision_on_change_column - unknown (unexpected)
--- PASS: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol - unknown (unexpected)
--- PASS: CompatibilityTest5_1#test_datetime_doesnt_set_precision_on_change_column - unknown (unexpected)
--- PASS: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default - unknown (unexpected)
--- PASS: CompatibilityTest6_1#test_options_are_not_validated - unknown (unexpected)
--- PASS: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz - unknown (unexpected)
--- PASS: CompatibilityTest6_0#test_options_are_not_validated - unknown (unexpected)
--- PASS: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime - unknown (unexpected)
--- PASS: CompatibilityTest5_0#test_options_are_not_validated - unknown (unexpected)
--- PASS: CompatibilityTest5_0#test_datetime_doesnt_set_precision_on_change_column - unknown (unexpected)
--- PASS: CompatibilityTest4_2#test_datetime_doesnt_set_precision_on_change_column - unknown (unexpected)
--- PASS: CompatibilityTest7_0#test_datetime_sets_precision_6_on_change_column - unknown (unexpected)
--- PASS: CompatibilityTest5_1#test_options_are_not_validated - unknown (unexpected)
--- PASS: CompatibilityTest4_2#test_options_are_not_validated - unknown (unexpected)
--- PASS: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default - unknown (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=false</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-17895302/activerecord/1732344149100/1732352508334)

</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

This test seems to be failing in each of the above runs:
```
2024/11/26 08:59:22 orm_helpers.go:135: --- FAIL: PostGISTest#test_point_to_json - unknown (expected)
```

--- a/pkg/cmd/roachtest/tests/activerecord_blocklist.go
+++ b/pkg/cmd/roachtest/tests/activerecord_blocklist.go
@@ -18,29 +18,7 @@ package tests
 // in the test log.
 
 var activeRecordBlocklist = blocklist{
-	`ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date`:                                               "unknown",
-	`ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array`:                                              "unknown",
-	`ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol`:                                             "unknown",
-	`ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime`:                              "unknown",
-	`ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default`:  "unknown",
-	`ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default`: "unknown",
-	`ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz`:                            "unknown",
-	`ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default`:                 "unknown",
-	`CompatibilityTest4_2#test_datetime_doesnt_set_precision_on_change_column`:                                                          "unknown",
-	`CompatibilityTest4_2#test_options_are_not_validated`:                                                                               "unknown",
-	`CompatibilityTest5_0#test_datetime_doesnt_set_precision_on_change_column`:                                                          "unknown",
-	`CompatibilityTest5_0#test_options_are_not_validated`:                                                                               "unknown",
-	`CompatibilityTest5_1#test_datetime_doesnt_set_precision_on_change_column`:                                                          "unknown",
-	`CompatibilityTest5_1#test_options_are_not_validated`:                                                                               "unknown",
-	`CompatibilityTest5_2#test_datetime_doesnt_set_precision_on_change_column`:                                                          "unknown",
-	`CompatibilityTest5_2#test_options_are_not_validated`:                                                                               "unknown",
-	`CompatibilityTest6_0#test_datetime_doesnt_set_precision_on_change_column`:                                                          "unknown",
-	`CompatibilityTest6_0#test_options_are_not_validated`:                                                                               "unknown",
-	`CompatibilityTest6_1#test_datetime_doesnt_set_precision_on_change_column`:                                                          "unknown",
-	`CompatibilityTest6_1#test_options_are_not_validated`:                                                                               "unknown",
-	`CompatibilityTest7_0#test_datetime_sets_precision_6_on_change_column`:                                                              "unknown",
-	`CompatibilityTest7_0#test_options_are_not_validated`:                                                                               "unknown",
-	`PostGISTest#test_point_to_json`:                                                                                                    "unknown",
+	`PostGISTest#test_point_to_json`: "unknown",
 }
 
 var activeRecordIgnoreList = blocklist{

